tests/ui/symbol-names/basic.rs

```diff
@@ -1,7 +1,7 @@
 //@ build-fail
 //@ revisions: legacy v0
 //@[legacy]compile-flags: -Z unstable-options -C symbol-mangling-version=legacy
-    //@[v0]compile-flags: -C symbol-mangling-version=v0
```

Comment:
Looking through history, it seems that the spaces were intended to align the `legacy` and `v0` arguments, but later one of them was given `-Z unstable-options` (probably via search/replace), so they ended up misaligned anyway.